internal/commands/sbomtest/summary.go

```diff
@@ -0,0 +1,44 @@
+package sbomtest
+
+import (
+	"encoding/json"
+
+	"github.com/snyk/go-application-framework/pkg/local_workflows/json_schemas"
+
+	"github.com/snyk/cli-extension-sbom/internal/snykclient"
+)
+
+type SummaryBuilder struct{}
```

Comment:
do we need a struct here? the method on it seems quite pure, it can be a simple function.

Reply:
you're right! I was thinking at the beginning that we might add some fields for customising, but doesn't really makes sense right now